.gitignore

```diff
@@ -18,3 +18,7 @@ cabal.sandbox.config
 .stack-work/
 cabal.project.local
 .HTF/
+H-Ray.cabal
+*~
+
+.idea/workspace.xml
```

Comment:
It's better to put whole `.idea` folder (and other Intellij IDEA specific files and folders) under `.gitignore` and remove completely from repository. Such files contains a lot of irrelevant changes (I see diff for 77 files with +4336 lines, this should never normally happen).